pkg/virtualnode/enqueue_requests_for_pod_events.go

```diff
@@ -0,0 +1,95 @@
+package virtualnode
```

Comment:
this should be in /pkg/cloudmap since it's used by the cloudmap controller.

Reply:
done